### ISSUE TITLE
Discussion: Moving to Gitlab?

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#1585]
**Summary:** 

Thanks to Microsoft is buying Github news, several projects are moving to Gitlab. Biggest handicap of moving Gitlab was, majority of projects and users were on Github. But current migration can change this situation and there is nothing that prevent us from migrating to Gitlab, a free software. 

What do you think team?

**Comment by nicolas-raoul** [comment]
While I preferred GitLab (which I use at work) to GitHub even before it was bought by Microsoft, requiring every contributor to create a GitLab account adds a barrier of entry.
I would be a bit like asking your friends to add you on Diaspora instead of Facebook: It is undeniably better, but almost nobody is there :-/

**Comment by nicolas-raoul** [comment]
GitHub->GitLab migration procedure: https://docs.gitlab.com/ee/user/project/import/github.html
For instance we would need to re-assign issues, it seems.

**Comment by ujjwalagrawal17** [comment]
I think GitHub is pretty stable. I am loving the new UI of GitHub. I think we don't have any plan to move to gitlab now. @josephine @nicolas-raoul @neslihanturan . If everyone is Ok we can close this. We have a lot of open issues (360).

**Comment by nicolas-raoul** [comment]
I think the topic is still open to discussion :-)

I don't have a problem with having a large number of open issues.
The important number is 67: It is the number of open bugs https://github.com/commons-app/apps-android-commons/issues?q=is%3Aissue+is%3Aopen+label%3Abug

**Comment by misaochan** [comment]
Yeah, I think it is OK to leave discussion issues open unless they become truly irrelevant (e.g. what to do for the 2018 hackathon which is already over). :)

**Comment by sivaraam** [comment]
Since this issue is related to GitLab and WMF is conducting a GitLab consultation, I thought I'll bring it to the notice of people here :)

[GitLab consultation - MediaWiki](https://www.mediawiki.org/wiki/GitLab_consultation)

TL;DR: WMF is conducting a consultation on replacing Gerrit with GitLab. It's open to comments now. They are open to comments until Sep 30.

**Comment by misaochan** [comment]
Thanks for the heads up @sivaraam !

**Comment by nicolas-raoul** [comment]
codeberg.org seems to be an option too, it looks exactly like GitHub ([example](https://codeberg.org/IzzyOnDroid/rbtlog/)).
